### PR TITLE
Check existence of DBI driver

### PR DESCRIPTION
**Pull request by outon**

### CODE REVIEW

File: check_sql/check_sql [hunk 1]
```diff
@@ -162,7 +162,12 @@ my $show = $np->opts->show;
 my ($label, $uom) = split(/,/, $np->opts->label);
 my $verbose = $np->opts->verbose;
 
-# TODO: Should check if the DBI driver exists
+# Check existence of DBI driver
+my @drivers = DBI->available_drivers;
+my ($DBI, $driver) = split /:/, $dsn;
+
+$np->nagios_exit('UNKNONW', 'DBI driver is required')
```
Comment (mr-russ) [inline check_sql/check_sql]: Unknown is spelt incorrectly, it would be useful to correct before a merge.

File: check_sql/check_sql [hunk 2]
```diff
@@ -162,7 +162,12 @@ my $show = $np->opts->show;
 my ($label, $uom) = split(/,/, $np->opts->label);
 my $verbose = $np->opts->verbose;
 
-# TODO: Should check if the DBI driver exists
+# Check existence of DBI driver
+my @drivers = DBI->available_drivers;
+my ($DBI, $driver) = split /:/, $dsn;
+
+$np->nagios_exit('UNKNOWN', 'DBI driver is required')
+   if (($DBI eq "DBI") && (join (", ", @drivers) !~ /$driver/));
```
Comment (danfruehauf) [inline check_sql/check_sql]: Please use a tab here for spacing, like is done in the rest of the file.

**Comment by danfruehauf** [comment]
Just one comment and please squash the commits together. Thanks for the contribution!

**Comment by danfruehauf** [comment]
Looks good. Can you please squash the commits together?

**Comment by danfruehauf** [comment]
Solved with 7b0ef18.

I did slight modifications to it. And thanks again for the contribution!

**Pull request closed without merging**